web-client/app/scripts/filters/timeToRead.js

```diff
@@ -3,6 +3,16 @@
 angular.module('webClientApp')
   .filter('timeToRead', [function () {
     return function(text) {
-      return Math.max(String(text).split(' ').length/200, 1);
+      var time =  Math.max(String(text).split(' ').length/200, 1).toFixed(0);
+      switch(true) {
```

Comment:
sorry for the inconvenience :worried: I thought switch will be more suitable for the multiple conditions. 
what about now ?